force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -160,9 +199,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
+        final Integer millisecondsPerMinute = 60000; // Make this value a constant
```

Comment:
When using the keyword final, it's best to move it outside of the method and put it in all caps.

ex.  static final Integer MILLISECONDS_PER_MINUTE = 6000
Try that and let me know if it still works